polaris-react/src/components/ActionMenu/components/MenuGroup/MenuGroup.tsx

```diff
@@ -12,6 +12,8 @@ export interface MenuGroupProps extends MenuGroupDescriptor {
   accessibilityLabel?: string;
   /** Whether or not the menu is open */
   active?: boolean;
+  /** Callback when the menu is clicked */
+  onClick?(openActions: () => void): void;
```

Comment:
Would you mind updating the description on the PR stating the rationale behind the change?

From what I can tell, this api change isn't needed. If we provide a `onClick` it'll be called with `handleOpen`. But all `handleOpen`  does it call `onOpen` with the title. I may be missing something, but can you not just use the `onOpen` prop?

Reply:
Currently there is no handler for the `actionGroup` title button. `onOpen` is not a public prop for the `actionGroups` it's defined on the parent component

Reply:
Sorry, just assumed this was a public component. Ignore me 😄